auraed/src/runtime/validation.rs

```diff
@@ -88,8 +89,8 @@ pub(crate) struct ValidatedCell {
     pub cpu_shares: u64,
     #[validate(none)]
     pub cpu_mems: String,
-    #[validate(none)]
-    pub cpu_quota: i64,
+    #[field_type(i64)]
+    pub cpu_quota: CpuQuota,
```

Comment:
The error is here. You need to add `#[validate(create)]` to automatically call `CpuQuota::validate_for_creation`